ui/pages/send/send-content/send-content.component.js

```diff
@@ -84,24 +88,52 @@ export default class SendContent extends Component {
     );
   }
 
+  checkContractAddress = () => {
+    const { recipient, tokenAddressList } = this.props;
+
+    getSymbolAndDecimals(recipient.userInput, tokenAddressList).then(
+      (result) => {
+        if (
+          (result.symbol !== undefined && result.symbol !== '') ||
```

Comment:
do we require both conditions here? Wouldn't  `result.symbol !== undefined || result.decimals !== undefined` be sufficient to establish that its on the list and what we would consider a know token/contract address?

Reply:
You have got the point.
Fixed in: ad39793344c75085bf0a75435319383fa7f7b514

Reply:
Actually, I had to modify this condition. The reason for doing so is the fact that entering some user's address gives the result: 
{ 
  symbol: '',
  decimals: '0'
}.
If the condition was just `result.symbol !== undefined || result.decimals !== undefined`, it would pass. 
Concerning that, I added another check, so the final condition looks like `if ((symbolAndDecimals.symbol !== undefined || symbolAndDecimals.decimals !== undefined) && symbolAndDecimals.symbol !== '')`.
If the address is contract, it will have the symbol as well.